HtmlTextView/src/main/java/org/sufficientlysecure/htmltextview/HtmlHttpImageGetter.java

```diff
@@ -125,10 +125,13 @@ protected void onPostExecute(Drawable result) {
             if (imageGetter == null) {
                 return;
             }
+
+            // re-set text to fix images overlapping text
+            CharSequence text = imageGetter.container.getText();
+            imageGetter.container.setText(text);
+
             // redraw the image by invalidating the container
             imageGetter.container.invalidate();
-            // re-set text to fix images overlapping text
-            imageGetter.container.setText(imageGetter.container.getText());
```

Comment:
Ok, no problem. This is stacktrace for ArrayOutOfBoundException, may be it's help:

```
Fatal Exception: java.lang.ArrayIndexOutOfBoundsException: length=116; index=852
       at android.text.MeasuredText.addStyleRun(MeasuredText.java:168)
       at android.text.MeasuredText.addStyleRun(MeasuredText.java:204)
       at android.text.StaticLayout.generate(StaticLayout.java:281)
       at android.text.DynamicLayout.reflow(DynamicLayout.java:284)
       at android.text.DynamicLayout.<init>(DynamicLayout.java:170)
       at android.widget.TextView.makeSingleLayout(TextView.java:5999)
       at android.widget.TextView.makeNewLayout(TextView.java:5897)
       at android.widget.TextView.checkForRelayout(TextView.java:6436)
       at android.widget.TextView.setText(TextView.java:3703)
       at android.widget.TextView.setText(TextView.java:3561)
       at android.widget.TextView.setText(TextView.java:3536)
       at org.sufficientlysecure.htmltextview.HtmlHttpImageGetter$ImageGetterAsyncTask.onPostExecute(HtmlHttpImageGetter.java:123)
       at org.sufficientlysecure.htmltextview.HtmlHttpImageGetter$ImageGetterAsyncTask.onPostExecute(HtmlHttpImageGetter.java:79)
       at android.os.AsyncTask.finish(AsyncTask.java:631)
       at android.os.AsyncTask.access$600(AsyncTask.java:177)
       at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:644)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:153)
       at android.app.ActivityThread.main(ActivityThread.java:5108)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:993)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:751)
       at dalvik.system.NativeStart.main(NativeStart.java)
```